alpha/template/composite/builder.go

```diff
@@ -28,13 +34,13 @@ type ContainerConfig struct {
 }
 
 type BuilderConfig struct {
-	ContainerCfg     ContainerConfig
-	OutputType       string
-	CurrentDirectory string
+	ContainerCfg   ContainerConfig
```

Comment:
Nit: Ideally we would simplify this by removing the `ContainerConfig` struct and instead just defining the one field we use from it directly in the `BuilderConfig` struct.